app/models/ValidationError.scala

```diff
@@ -39,6 +39,11 @@ final case class MismatchedPage(query: Query) extends ValidationError {
   val message: String = s"Mismatched page: ${query.path}"
 }
 
+final case class IncorrectlyAnsweredPage(query: Query) extends ValidationError {
```

Comment:
fancy new validation error for when you end up on a cya page you shouldn't be on